src/mimeparser.rs

```diff
@@ -1,19 +1,19 @@
 use std::collections::{HashMap, HashSet};
 
+use anyhow::Context as _;
```

Comment:
What's this for?

Reply:
importing the trait, and avoiding to avoid overlapping with `context::Context`